### ISSUE TITLE
False positive "Commenting.InlineDocCommentDeclaration.InvalidFormat" when used with PHP8 attributes

### ISSUE THREAD

**Issue by alekitto** [slevomat/coding-standard#1205]
Hi and thanks for your work on this, I found it very useful in many projects.  
I think I found a false positive when using `@var` doc comment on a class property with PHP8 attributes:

```php
class Test {
    /** @var string */    // <- This is marked as an error
    #[Assert\NotBlank]
    public $text;

    #[Assert\NotBlank]
    /** @var string */    // <- This is OK
    public $foo;
}
```

The two formats should be equally valid, but on the first one an error is raised stating that:

```
Invalid inline documentation comment format "@var string", expected "@var type $variable". (SlevomatCodingStandard.Commenting.InlineDocCommentDeclaration.InvalidFormat)
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/922e94689575f1d674c2c2369b6c1f130b8bfa1d

**Comment by temp** [comment]
We are still experiencing this bug with 7.0.7, which contains the bugfix.

```
    /** @var DamageInput[]|null */
    #[Groups(['read', 'write'])]
    #[Valid]
    public ?array $damages = null;
```

```
 20 | ERROR | Invalid inline documentation comment format "@var DamageInput[]|null", expected "@var type $variable".
    |       | (SlevomatCodingStandard.Commenting.InlineDocCommentDeclaration.InvalidFormat)
```

Is there a preferred order of docblock/attributes? phpcs fails with this variant, phpstan the other way round.

**Comment by kukulich** [comment]
@temp Please open new issue.

**Comment by temp** [comment]
Will do, thanks
